packages/web-components/@sfx/search-box/README.md

```diff
@@ -0,0 +1,46 @@
+# Sf-X Search Box Component
+
+## Functionality
+
+The component accepts text input and dispatches events based on input.
+
+### Events
+
+#### `sfx::searchbox_hover_event`
+
+Dispatched when a user hovers over the search box.
+
+#### `sfx::searchbox_click_event`
+
+Dispatched when a user clicks anywhere within the search box input area.
+
+#### `sfx::search_request`
+
+Dispatched when a user clicks on the search button on hits `enter` within the search box. This event sends the search term value entered into the search box.
+
+#### `sfx::search_box_cleared`
+
+Dispatched when the value within the search box area is cleared. Occurs when a user clicks on the clear button, or when the final letter is removed from the search term.
+
+#### `sfx::autocomplete_request`
+
+Dispatched when the search term value within the search box contains more than 3 characters. This event sends the search term value entered into the search box.
```

Comment:
Shouldn't there be a way for users to configure at what amount of characters the autocomplete request goes out? And by extension doesn't that mean that this event will trigger based on that configured number?

Reply:
Discussion with Johann lead to the conclusion that this should be handled by the component driver. I've removed all autocomplete events from this PR.